scripts/variantstore/wdl/test/GvsTieOutVDS.wdl

```diff
@@ -0,0 +1,130 @@
+version 1.0
```

Comment:
this is mostly refactored from existing code to be able to be called as a standalone WDL